download_artifacts/download_artifacts.py

```diff
@@ -0,0 +1,65 @@
+import json
+import re
+import sys
+
+import requests
+
+from global_config import DAConfig
+from nexusURL import NexusURL
+
+VERSION_MATCH = r"<version>(.*?)(|-SNAPSHOT)</version>"
```

Comment:
VERSION_REGEX
NAME_REGEX

For Literals, use ENUM:
Refer here: https://github.com/confluentinc/release-tools/blob/master/confluent/release/constants.py

You might not want to expose these REGEX and LITERAL constants as public, instead encapsulate them within the class.